test/unit/clients/connector_client/connector_patch_apple_pay_toggle_test.js

```diff
@@ -0,0 +1,93 @@
+'use strict'
+
+// NPM dependencies
+const Pact = require('pact')
+const chai = require('chai')
+const chaiAsPromised = require('chai-as-promised')
+
+// Custom dependencies
+const path = require('path')
+const PactInteractionBuilder = require('../../../fixtures/pact_interaction_builder').PactInteractionBuilder
+const Connector = require('../../../../app/services/clients/connector_client').ConnectorClient
+const gatewayAccountFixtures = require('../../../fixtures/gateway_account_fixtures')
+
+// Constants
+const ACCOUNTS_RESOURCE = '/v1/api/accounts'
+const port = Math.floor(Math.random() * 48127) + 1024
+const connectorClient = new Connector(`http://localhost:${port}`)
+const expect = chai.expect
+const existingGatewayAccountId = 666
+
+// Global setup
+chai.use(chaiAsPromised)
+
+describe('connector client - patch apple pay toggle (enabled) request', () => {
+  const request = gatewayAccountFixtures.validGatewayAccountApplePayToggleRequest(true).getPlain()
+  let provider = Pact({
+    consumer: 'selfservice',
+    provider: 'connector',
+    port: port,
+    log: path.resolve(process.cwd(), 'logs', 'mockserver-integration.log'),
+    dir: path.resolve(process.cwd(), 'pacts'),
+    spec: 2,
+    pactfileWriteMode: 'merge'
+  })
+
+  before(() => provider.setup())
+  after(() => provider.finalize())
+
+  describe('apple pay toggle - supported payment provider request', () => {
+    const applePayToggleSupportedPaymentProvider =
```

Comment:
perhaps this should end in `State` as per the other similar names.

Reply:
👍 Good shout. Thanks